<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>360Â° Panorama Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="image_label">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="start_recording_btn">
        <property name="text">
         <string>Start Recording</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_recording_btn">
        <property name="text">
         <string>Stop Recording</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_still_btn">
        <property name="text">
         <string>Save Still</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="filter_combo">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grayscale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sepia</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="apply_filter_btn">
        <property name="text">
         <string>Apply Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
